assets/tests/components/searchResults.test.tsx

```diff
@@ -285,3 +423,34 @@ describe("SearchResults", () => {
     expect(testDispatch).toHaveBeenCalledWith(setSearchText(""))
   })
 })
+
+describe("byOperatorLogonTime", () => {
+  test("sorts more recent logons ahead of less recent ones", () => {
+    const oldVehicle: Vehicle = {
+      id: "1",
+      operatorLogonTime: new Date("2018-08-15T13:30:00.000Z"),
+    } as Vehicle
+    const newVehicle: Vehicle = {
+      id: "2",
+      operatorLogonTime: new Date("2018-08-15T13:40:00.000Z"),
+    } as Vehicle
+
+    expect(byOperatorLogonTime(oldVehicle, newVehicle)).toEqual(1)
+    expect(byOperatorLogonTime(newVehicle, oldVehicle)).toEqual(-1)
+    expect(byOperatorLogonTime(newVehicle, newVehicle)).toEqual(0)
+  })
+
+  test("sorts ghosts ahead of vehicles", () => {
+    const vehicle: Vehicle = {
+      id: "1",
+      operatorLogonTime: new Date("2018-08-15T13:30:00.000Z"),
+    } as Vehicle
+    const ghost: Vehicle = {
+      id: "ghost-2",
+    } as Vehicle
+
+    expect(byOperatorLogonTime(vehicle, ghost)).toEqual(1)
+    expect(byOperatorLogonTime(ghost, vehicle)).toEqual(-1)
+    expect(byOperatorLogonTime(ghost, ghost)).toEqual(0)
```

Comment:
I know we had a bug a while ago where the `-1` and `+1` were backwards. It's hard to know by looking at these whether the test is correct.

What if these tests were instead something like
```ts
expect([vehicle, ghost].sort(byOperatorLogonTime)).toEqual([ghost, vehicle])
```
where the test is based on easy to understand list ordering?

Reply:
Yeah, I like that much better.